specification/conformance/conformance_rules/columns/consumedunit.json

```
@@ -0,0 +1,195 @@
+      "Requirement": {
+        "CheckFunction": "AND",
+        "Items": [
+          {
+            "CheckFunction": "CheckConformanceRule",
+            "ConformanceRuleId": "ConsumedUnit-C-001-M"
+          },
+          {
+            "CheckFunction": "CheckConformanceRule",
+            "ConformanceRuleId": "ConsumedUnit-C-002-M"
+          },
+          {
+            "CheckFunction": "CheckConformanceRule",
+            "ConformanceRuleId": "ConsumedUnit-C-003-O"
+          },
+          {
+            "CheckFunction": "CheckConformanceRule",
+            "ConformanceRuleId": "ConsumedUnit-C-004-C"
+          }
+        ]
+      },
+      "Condition": {},
+      "Dependencies": [
+        "CostAndUsage-D-066-C",
+        "ConsumedUnit-C-001-M",
+        "ConsumedUnit-C-002-M",
+        "ConsumedUnit-C-003-O",
+        "ConsumedUnit-C-004-C"
+      ]
+    }
+  },
+  "ConsumedUnit-C-001-M": {
+    "Function": "Type",
+    "Reference": "ConsumedUnit",
+    "EntityType": "Column",
+    "Notes": "",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "ApplicabilityCriteria": [],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "ConsumedUnit MUST be of type String.",
+      "Keyword": "MUST",
+      "Requirement": {
+        "CheckFunction": "TypeString",
+        "ColumnName": "ConsumedUnit"
+      },
+      "Condition": {},
+      "Dependencies": []
+    }
+  },
+  "ConsumedUnit-C-002-M": {
+    "Function": "Format",
+    "Reference": "ConsumedUnit",
+    "EntityType": "Column",
+    "Notes": "",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "ApplicabilityCriteria": [],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "ConsumedUnit MUST conform to StringHandling requirements.",
+      "Keyword": "MUST",
+      "Requirement": {
+        "CheckFunction": "FormatString",
+        "ColumnName": "ConsumedUnit"
+      },
+      "Condition": {},
+      "Dependencies": []
+    }
+  },
+  "ConsumedUnit-C-003-O": {
+    "Function": "Format",
+    "Reference": "ConsumedUnit",
+    "EntityType": "Column",
+    "Notes": "",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "ApplicabilityCriteria": [],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "ConsumedUnit SHOULD conform to UnitFormat requirements.",
+      "Keyword": "SHOULD",
+      "Requirement": {
+        "CheckFunction": "FormatUnit",
+        "ColumnName": "ConsumedUnit"
+      },
+      "Condition": {},
+      "Dependencies": []
+    }
+  },
+  "ConsumedUnit-C-004-C": {
+    "Function": "Composite",
+    "Reference": "ConsumedUnit",
+    "EntityType": "Column",
+    "Notes": "",
+    "CRVersionIntroduced": "1.2",
+    "Status": "Active",
+    "ApplicabilityCriteria": [
+      "USAGE_MEASUREMENT_SUPPORTED"
+    ],
+    "Type": "Static",
+    "ValidationCriteria": {
+      "MustSatisfy": "The ConsumedUnit column adheres to the following requirements:",
```

All (in this case, both) elements in this composite function are related to nullability, so the following pattern applies:

> "MustSatisfy": "{ColumnId} nullability is defined as follows:" 

```suggestion
      "MustSatisfy": "ConsumedUnit nullability is defined as follows:",
```